From 0821066f9adf8410891cd07684ecac50a9bc36a4 Mon Sep 17 00:00:00 2001
From: Fabio Porcedda <fabio.porcedda@gmail>
Date: Wed, 25 Feb 2015 18:40:24 +0100
Subject: [PATCH] Lua: fix a typo changing %(LFS_DIR) to $(LFS_DIR)

Also this fix a error on GNU Make v4.0:
  Makefile:203: *** mixed implicit and normal rules.  Stop.

Signed-off-by: Fabio Porcedda <fabio.porcedda@gmail>
---
 resources/Makefile.in-lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/resources/Makefile.in-lua b/resources/Makefile.in-lua
index 0902f11..f3d95fd 100644
--- a/resources/Makefile.in-lua
+++ b/resources/Makefile.in-lua
@@ -57,5 +57,5 @@ LFS_CFLAGS = -I$(LFS_DIR)
 
 OBJECTS += $(LUA_OBJECTS) $(SQLITE_OBJECTS) $(LFS_OBJECTS)
 CFLAGS += $(LUA_CFLAGS) $(SQLITE_CFLAGS) $(LFS_CFLAGS) -DUSE_LUA -DUSE_LUA_SQLITE3 -DUSE_LUA_FILE_SYSTEM
-SOURCE_DIRS = $(LUA_DIR) $(SQLITE_DIR) %(LFS_DIR)
+SOURCE_DIRS = $(LUA_DIR) $(SQLITE_DIR) $(LFS_DIR)
 
-- 
2.3.0

